inc/class-wpseo-admin-bar-menu.php

```diff
@@ -76,6 +76,11 @@ public function __construct( WPSEO_Admin_Asset_Manager $asset_manager = null ) {
 	 */
 	public function add_menu( WP_Admin_Bar $wp_admin_bar ) {
 
+		// On block editor pages, the admin bar only shows on mobile, where having this menu icon is not very helpful.
+		if ( get_current_screen()->is_block_editor() ) {
```

Comment:
Shouldn't we do a `null` check because `get_current_screen()` can return `null`

Reply:
I think you're right @andizer, thx :)